boards/cc1352p-launchpad/Makefile.dep

```diff
@@ -0,0 +1,5 @@
+ifneq (,$(filter gnrc_netdev_default,$(USEMODULE)))
+  USEMODULE += netif
+  USEMODULE += cc26x2_cc13x2_rf
+  USEMODULE += netdev_ieee802154
```

Comment:
Only `cc26x2_cc13x2_rf` should be needed here - but that can be added together with the radio driver.

Reply:
The radio driver is almost ready. We are  running some stress tests before pushing changes.
If that's possible i would like to change it when the radio driver merges

Reply:
It will still take some time to review the radio driver, whereas the board could already be merged now if the remaining comments are addressed.

Reply:
Ok, so i remove this file for now and it will be added later with the radio driver.